addons/io/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/api/HueState.java

```diff
@@ -30,6 +30,10 @@
     public String colormode = "ct";
     public boolean reachable = true;
 
+    public static short convertBrightness(int percentBrightness) {
+      return (short) Math.floor((double)(percentBrightness  * 255) / 100d);
```

Comment:
Fix indentation and remove unneccesary space:

```suggestion
        return (short) Math.floor((double)(percentBrightness * 255) / 100d);
```

Reply:
No I can not.
After 1 year it's way too late.

The code is even not up to date anymore